logging/subscriber/src/subscriber.ts

```diff
@@ -0,0 +1,37 @@
+import type { EventPayload, Layer, Level } from '@rango-dev/logging-types';
+
+import { EventType } from '@rango-dev/logging-types';
+
+import { isEnabled, isValidEvent } from './helpers';
+
+interface Options {
+  baseLevel: Level;
+}
+
+export function init(layers: Layer[], options: Options) {
+  const { baseLevel } = options;
+
+  document.addEventListener(EventType, function handler(e) {
```

Comment:
I think we need to clean up listeners.

Reply:
This should be called once, during the whole app lifetime. Giving that option to clean up the event will be misleading I believe. it can causes to miss events (before init and after clean up) if do something rather than this way and it will be unreliable.